### ISSUE TITLE
PGNet training configs

### ISSUE THREAD

**Issue by Daisy5296** [PaddlePaddle/PaddleOCR#10126]
请提供下述完整信息以便快速定位问题/Please provide the following information to quickly locate the problem

- 系统环境/System Environment：
- 版本号/Version：Paddle：  PaddleOCR： 问题相关组件/Related components：
- 运行指令/Command Code：
- 完整报错/Complete Error Message：

Hi I am training the PGNet from scratch using 20w+data containing chinese, english and digits. The cost reduced nicely at the begining and seemed to slow down and maybe unchanged in epoch10-12. The  loss is like:

[2023/06/09 09:56:30] ppocr INFO: epoch: [12/600], global_step: 41880, lr: 0.000234, loss: 0.277034, score_loss: 0.041784, border_loss: 0.005397, direction_loss: 0.002723, ctc_loss: 0.042630, avg_reader_cost: 0.05683 s, avg_batch_cost: 1.34881 s, avg_samples: 14.0, ips: 10.37954 samples/s, eta: 39 days, 16:11:22

It seems to be small enough, is that correct? But the evaluation seems to be not very good:

[2023/06/09 08:48:26] ppocr INFO: best metric, f_score_e2e: 0, is_float16: False, total_num_gt: 11000, total_num_det: 1602, global_accumulative_recall: 781.5999999999983, hit_str_count: 0, recall: 0.0710545454545453, precision: 0.6087390761548075, f_score: 0.12725532273809906, seqerr: 1.0, recall_e2e: 0.0, precision_e2e: 0.0, fps: 38.13785620010186, best_epoch: 11

Could you please share some of your experience in training this network. Shall I continue this training process, or change the learning rate to restart, or any other improvements to do? The configs I used are:

Global:
  use_gpu: True
  epoch_num: 600
  log_smooth_window: 20
  print_batch_step: 10
  save_model_dir: ./output/pgnet_test/
  save_epoch_step: 1 #10
  eval_batch_step: [ 0, 3585 ]  #[0,1000]
  cal_metric_during_train: False
  pretrained_model:
  checkpoints:
  save_inference_dir:
  use_visualdl: False
  infer_img:
  infer_visual_type: CN # two mode: EN is for english datasets, CN is for chinese datasets
  valid_set: partvgg # two mode: totaltext valid curved words, partvgg valid non-curved words
  save_res_path: ./output/pgnet_r50_vd_totaltext/predicts_pgnet.txt
  character_dict_path: /dfs/data/data/plate/data/dict_CCPD.txt
  character_type: CN
  max_text_length: 10 # the max length in seq
  max_text_nums: 10 # the max seq nums in a pic
  tcl_len: 64

Architecture:
  model_type: e2e
  algorithm: PGNet
  Transform:
  Backbone:
    name: ResNet
    layers: 50
  Neck:
    name: PGFPN
  Head:
    name: PGHead
    character_dict_path: /dfs/data/data/plate/data/dict_CCPD.txt # the same as Global:character_dict_path

Loss:
  name: PGLoss
  tcl_bs: 64
  max_text_length: 10 # the same as Global: max_text_length
  max_text_nums: 10 # the same as Global：max_text_nums
  pad_num: 65 # the length of dict for pad

Optimizer:
  name: Adam
  beta1: 0.9
  beta2: 0.999
  lr:
    name: Cosine
    learning_rate: 0.001
    warmup_epoch: 50
  regularizer:
    name: 'L2'
    factor: 0.0001

PostProcess:
  name: PGPostProcess
  score_thresh: 0.5
  mode: fast   # fast or slow two ways
  point_gather_mode: align # same as PGProcessTrain: point_gather_mode

Metric:
  name: E2EMetric
  mode: A   # two ways for eval, A: label from txt,  B: label from gt_mat
  gt_mat_dir:  ./train_data/total_text/gt  # the dir of gt_mat
  character_dict_path: /dfs/data/data/plate/data/dict_CCPD.txt
  main_indicator: f_score_e2e

Train:
  dataset:
    name: PGDataSet
    data_dir: /dfs/data/data/plate/data/
    label_file_list: [/dfs/data/data/plate/data/train_CCPD2019base_CCPD2020trainTest.txt]
    ratio_list: [1.0]
    transforms:
      - DecodeImage: # load image
          img_mode: BGR
          channel_first: False
      - E2ELabelEncodeTrain:
      - PGProcessTrain:
          batch_size: 14  # same as loader: batch_size_per_card
          use_resize: True
          use_random_crop: False
          min_crop_size: 24
          min_text_size: 4
          max_text_size: 512
          point_gather_mode: align # two mode: align and none, align mode is better than none mode
      - KeepKeys:
          keep_keys: [ 'images', 'tcl_maps', 'tcl_label_maps', 'border_maps','direction_maps', 'training_masks', 'label_list', 'pos_list', 'pos_mask' ] # dataloader will return list in this order
  loader:
    shuffle: True
    drop_last: True
    batch_size_per_card: 14
    num_workers: 16

Eval:
  dataset:
    name: PGDataSet
    data_dir: /dfs/data/data/plate/data
    label_file_list: [/dfs/data/data/plate/data/val_CCPD2019base_CCPD2020val.txt]
    transforms:
      - DecodeImage: # load image
          img_mode: BGR
          channel_first: False
      - E2ELabelEncodeTest:
      - E2EResizeForTest:
          max_side_len: 768
      - NormalizeImage:
          scale: 1./255.
          mean: [ 0.485, 0.456, 0.406 ]
          std: [ 0.229, 0.224, 0.225 ]
          order: 'hwc'
      - ToCHWImage:
      - KeepKeys:
          keep_keys: [ 'image', 'shape', 'polys', 'texts', 'ignore_tags', 'img_id']
  loader:
    shuffle: False
    drop_last: False
    batch_size_per_card: 1 # must be 1
    num_workers: 2
